src/core_plugins/kibana/server/tutorials/netflow/index.js

```diff
@@ -0,0 +1,66 @@
+import { TUTORIAL_CATEGORY } from '../../../common/tutorials/tutorial_category';
+import { INSTRUCTION_VARIANT } from '../../../common/tutorials/instruction_variant';
+
+export function netflowSpecProvider() {
+  return {
+    id: 'netflow',
+    name: 'Netflow',
+    category: TUTORIAL_CATEGORY.SECURITY,
+    shortDescription: 'The Logstash Netflow module simplifies the collection, normalization, and visualization of network flow data',
+    longDescription: 'The Logstash Netflow module simplifies the collection, normalization, and visualization of network flow data.' +
+      ' With a single command, the module parses network flow data, indexes the events into Elasticsearch, and installs a suite of Kibana' +
+      ' dashboards to get you exploring your data immediately. Logstash modules support Netflow Version 5 and 9.',
+    //iconPath: '', TODO
+    completionTimeMinutes: 10,
+    //previewImagePath: 'kibana-apache.png', TODO
+    params: [
+      {
+        'netflow.var.input.udp.port': {
+          type: 'number', // TODO: Make this a const as well?
```

Comment:
@nreese What do you think of making these consts/enums as well?

Reply:
good idea. I have started a new file for param type constants that you can use - https://github.com/nreese/kibana/blob/home_phase_two/src/core_plugins/kibana/common/tutorials/param_types.js